### PR TITLE
enable setWritablePath on platform win32 and mac

### PR DESCRIPTION
**Pull request by kpkhxlgy0**

### CODE REVIEW

File: cocos/platform/apple/CCFileUtils-apple.mm [hunk 1]
```diff
@@ -337,6 +337,11 @@ static void addObjectToNSDict(const std::string& key, const Value& value, NSMuta
 
 std::string FileUtilsApple::getWritablePath() const
 {
+    if (_writablePath.length())
```
Comment (WenhaiLin) [inline cocos/platform/apple/CCFileUtils-apple.mm:340]: I suggest you determines whether or not the string is not empty like this:

``` cpp
if ( !_writablePath.empty())
```

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by minggo** [comment]
@kpkhxlgy0 
I don't find where to enable setWritablePath.

**Comment by kpkhxlgy0** [comment]
@minggo 
I call setWritablePath, but there's no effect on mac and win32, the getWritablePath() always return the same path.

**Comment by minggo** [comment]
@kpkhxlgy0 
Got it thanks.

**Comment by minggo** [comment]
[ci rebuild]

**Pull request merged**